### ISSUE TITLE
Quotes are removed from date-time example field in openapi.yaml

### ISSUE THREAD

**Issue by srnjak** [OpenLiberty/open-liberty#12722]
I created openapi.yaml by hand and creating an example for string field which holds date-time. The value (i.e.: `2020-06-20T09:29:00Z`) has to be in quotes. After accessing the openapi specification from `http://localhost:9080/openapi`, there is value without quotes.

Here are examples:
_openapi.yaml_
```
openapi: 3.0.0
info:
  title: test
  version: '1.0'
paths:
  /api/test:
    post:
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/MySchema'
      responses:
        default:
          description: default response
components:
  schemas:
    MySchema:
      type: object
      properties:
        started:
          type: string
          format: date-time
          example: '2020-06-20T09:29:00Z'
        color:
          type: string
          example: '#34f4ba'
```
_Outupt from http://localhost:9080/openapi_
```
openapi: 3.0.0
info:
  title: test
  version: "1.0"
servers:
- url: http://localhost:9081/rest
paths:
  /api/test:
    post:
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/MySchema'
      responses:
        default:
          description: default response
components:
  schemas:
    MySchema:
      type: object
      properties:
        started:
          type: string
          format: date-time
          example: 2020-06-20T09:29:00Z
        color:
          type: string
          example: '#34f4ba'
```
Consequently in UI there is written `{}` instead of the date/time.
![image](https://user-images.githubusercontent.com/4569416/85202080-380f7680-b304-11ea-820b-c98654877138.png)

OpenApi endpoint should not omit the quotes surrounding the date/time value.

Expected output would be:
```
openapi: 3.0.0
info:
  title: test
  version: "1.0"
servers:
- url: http://localhost:9080/rest
paths:
  /api/test:
    post:
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/MySchema'
      responses:
        default:
          description: default response
components:
  schemas:
    MySchema:
      type: object
      properties:
        started:
          type: string
          format: date-time
          example: '2020-06-20T09:29:00Z'
        color:
          type: string
          example: '#34f4ba'
```

 - OpenLiberty Version: 20.0.0.6

**Comment by msmiths** [comment]
This does not appear to be a problem with the latest version of the `mpOpenAPi` feature, i.e., `mpOpenAPI-2.0`. The OpenAPI document that is served by http://localhost:9080/openapi is as follows:
```
---
openapi: 3.0.0
info:
  title: test
  version: "1.0"
servers:
- url: http://localhost:9080
- url: https://localhost:9443
paths:
  /api/test:
    post:
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/MySchema'
      responses:
        default:
          description: default response
components:
  schemas:
    MySchema:
      type: object
      properties:
        started:
          format: date-time
          type: string
          example: 2020-06-20T09:29:00Z
        color:
          type: string
          example: '#34f4ba'
```
Notice that the quotes are still removed from the `example` for the `started` property.

However, the generated JSON for the POST method in Swagger UI is as follows:

<img width="379" alt="image" src="https://user-images.githubusercontent.com/1774315/118634042-05972380-b7ca-11eb-9d85-aeebdf472078.png">

Notice that the value of the `started` property matches the one specified in the example.

**Comment by srnjak** [comment]
I can confirm that issue is resolved in `mpOpenAPI-2.0` using OpenLiberty `21.0.0.3`.
